### PR TITLE
chore(updatecli): update manifest to new actions

### PR DESCRIPTION
**Pull request by smerleCB**

### CODE REVIEW

File: updatecli/updatecli.d/terraform-providers.yaml [hunk 1]
```diff
@@ -27,12 +27,11 @@ targets:
...
 
-pullrequests:
+actions:
   default:
-    kind: github
-    scmID: default
-    targets:
-      - upgradeActualLockFile
+    kind: github/pullrequest
+    scmid: default
```
Comment (lemeurherve) [inline updatecli/updatecli.d/terraform-providers.yaml:33]:
```suggestion
    scmid: default
    title: "Bump Terraform plugins - not modules"
```
Also, `title` at line 1 have to be renamed `name`

File: updatecli/updatecli.d/terraform-providers.yaml [hunk 2]
```diff
@@ -27,12 +27,11 @@ targets:
...
     spec:
       labels:
         - dependencies
+        - terraform-providers
```
Comment (lemeurherve) [inline updatecli/updatecli.d/terraform-providers.yaml:37]:
```suggestion
        - terraform-providers

```
Missing EOL

File: updatecli/updatecli.d/terraform-providers.yaml (hunk 1 above)
Comment (lemeurherve) [reply updatecli/updatecli.d/terraform-providers.yaml:33]: To be tested but the renaming might be the only needed change.

Comment (smerleCB) [reply updatecli/updatecli.d/terraform-providers.yaml:33]: I would says so as I took example on this one : https://github.com/jenkins-infra/azure/blob/main/updatecli/updatecli.d/terraform-providers.yaml

File: updatecli/updatecli.d/terraform-providers.yaml (hunk 2 above)
Comment (lemeurherve) [reply updatecli/updatecli.d/terraform-providers.yaml:37]: You can see the absence of End Of File in github diff:
Screenshot 2023-06-23 at 15 25 49

The other files have an EOL.

**Review by dduportal — APPROVED** [review]
Build is OK 👍 go go go

**Pull request merged**